docs/src/main/paradox/index.md

```diff
@@ -1,273 +1,93 @@
+    from the root directory, enter: `cd akka-distributed-workers-scala`
 
-Take a look at the logging that is done in 
-<a href="#code/src/main/scala/worker/WorkProducer.scala" class="shortcut">WorkProducer.scala</a>,
-<a href="#code/src/main/scala/worker/Master.scala" class="shortcut">Master</a>
-and <a href="#code/src/main/scala/worker/Worker.scala" class="shortcut">Worker.scala</a>.
-Identify the corresponding log entries in the 3 terminal windows.
+1. Enter `./sbt` on OSX/Linux or `sbt.bat` on Windows to start sbt.
+ 
+    sbt downloads project dependencies. The `>` prompt indicates sbt has started in interactive mode.
 
-Shutdown the worker node (third terminal window) with <code>ctrl-c</code>.
-Observe how the <code>"Consumed result"</code> logs in the frontend node (second terminal window)
-stops. Start the worker node again.
+1. At the sbt prompt, enter `run`.
+ 
+    sbt builds the project and runs the `Main` of the project:
 
-```bash
-sbt "runMain worker.Main 0"
+After waiting a few seconds for the cluster to form the output should start look _something_ like this (scroll all the way to the right to see the Actor output):
+ 
 ```
-
-You can also start more such worker nodes in new terminal windows.
-
-You can start more cluster backend nodes using port numbers between 2000-2999:
-
-```bash
-sbt "runMain worker.Main 2552"
+[INFO] [07/21/2017 17:41:53.320] [ClusterSystem-akka.actor.default-dispatcher-16] [akka.tcp://ClusterSystem@127.0.0.1:51983/user/producer] Produced work: 3
+[INFO] [07/21/2017 17:41:53.322] [ClusterSystem-akka.actor.default-dispatcher-3] [akka.tcp://ClusterSystem@127.0.0.1:2551/user/master/singleton] Accepted work: 3bce4d6d-eaae-4da6-b316-0c6f566f2399
+[INFO] [07/21/2017 17:41:53.328] [ClusterSystem-akka.actor.default-dispatcher-3] [akka.tcp://ClusterSystem@127.0.0.1:2551/user/master/singleton] Giving worker 2b646020-6273-437c-aa0d-4aad6f12fb47 some work 3bce4d6d-eaae-4da6-b316-0c6f566f2399
+[INFO] [07/21/2017 17:41:53.328] [ClusterSystem-akka.actor.default-dispatcher-2] [akka.tcp://ClusterSystem@127.0.0.1:51980/user/worker] Got work: 3
+[INFO] [07/21/2017 17:41:53.328] [ClusterSystem-akka.actor.default-dispatcher-16] [akka.tcp://ClusterSystem@127.0.0.1:51980/user/worker] Work is complete. Result 3 * 3 = 9.
+[INFO] [07/21/2017 17:41:53.329] [ClusterSystem-akka.actor.default-dispatcher-19] [akka.tcp://ClusterSystem@127.0.0.1:2551/user/master/singleton] Work 3bce4d6d-eaae-4da6-b316-0c6f566f2399 is done by worker 2b646020-6273-437c-aa0d-4aad6f12fb47
 ```
+   
+Congratulations, you just ran your first Akka Cluster app. Now take a look at what happened under the covers. 
 
-You can start more cluster frontend nodes using port numbers between 3000-3999:
+## What happens when you run it
 
-```
-sbt "runMain worker.Main 3002"		
-```
+When `Main` is run without any parameters, it starts 6 `ActorSystem`s in the same JVM which then form a single cluster. 
```

Comment:
suggest: When `Main` runs without any parameters, it starts six `ActorSystem`s in the same JVM. These six `ActorSystem`s  form a single cluster. The six nodes include two each that perform front-end, back-end, and worker tasks:
- The front-end nodes simulate an external interface, such as a REST API, that accepts workloads from clients.
- The worker nodes have worker actors that accept and process workloads.
- The back-end nodes contain a `Master` actor that coordinates workloads, keeps track of the workers, and delegates work to available workers. One of the nodes is active and one is on standby. If the active `Master` goes down, the standby takes over. 

(If this wording works for you, you can delete the three paragraphs below)